hack/lint-generate.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
```

Comment:
I was tripped up trying to figure out what this line did. I am still not 100% sure why it was needed in the first place. This started in [run-wmcb-ci-e2e-test.sh#L25](https://github.com/openshift/windows-machine-config-bootstrapper/blob/master/hack/run-wmcb-ci-e2e-test.sh#L25) and I would hate for it to propagated more if it can be avoided. I prefer using `pwd` if that works like in [run-wsu-ci-e2e-test.sh#L25](https://github.com/openshift/windows-machine-config-bootstrapper/blob/master/hack/run-wsu-ci-e2e-test.sh#L25)